### PR TITLE
Add cleanup and uninstall functions

### LINKED ISSUE #21: `cleanup` function in the core code
There is a **clean.sh** file in the repo. As the package creates many files in projects folder and julia bin directory, it would be nice to have a `cleanup` routine in the code as a **functionality** of the package.

Example : 
```
julia> using AutoSysimages
julia> AutoSysimages.cleanup()
```
Should remove all the files and directory created by the package, including the asysimg script

### PR DESCRIPTION
**Pull request by petvana**
It implements

- `uninstall()`
  - Removes what `install()` created **and call `cleanup()`**
- `cleanup()`
  - Removes directory `joinpath(DEPOT_PATH[1], "asysimg")`
  - ~~Removes all `SysimagePreferences.toml` files~~

Both function has optional named arguments `(;verbose::Bool = true, dry::Bool = false)`.

Also, it ~~fixes `julia_args.jl` for the case of non-existing active directory and~~ removes `clean.sh` script.

### CODE REVIEW

File: src/AutoSysimages.jl [hunk 1]
```diff
@@ -389,8 +390,7 @@ end
 """
```
Comment (curio-sitas) [inline src/AutoSysimages.jl:399]: One can be more specific on the warnings :
```julia
if isnothing(dir)
        @warn """Install directory not found."""
        return
end

if !(Sys.iswindows() || Sys.isunix())
        @warn """AutoSysimages: Installation is not yet supported for your OS.
Feel free to submit a PR."""
        return
    end
```

File: src/AutoSysimages.jl [hunk 2]
```diff
@@ -444,6 +442,55 @@ If this not works please add the script into your `PATH`."""
 
 end
 
+"""
+    uninstall(;verbose::Bool = true, dry::Bool = false)
+
+This removes the `asysimg` scripts and all the generated files (by calling `cleanup`).
+"""
+function uninstall(dir = _default_install_dir();verbose::Bool = true, dry::Bool = false)
+    file_name = Sys.iswindows() ? "asysimg.bat" : "asysimg"
```
Comment (curio-sitas) [inline src/AutoSysimages.jl:464]: Check if the default or specified install dir exists

```julia
if isnothing(dir)
        @warn """Install directory not found."""
        return
end
```

**Review by curio-sitas — COMMENTED** [review]
Maybe we could keep trace of all packages with Preferences.toml files (added when building sysimages) in a file and run over that list for the cleaning ?

**Comment by petvana** [comment]
> Maybe we could keep trace of all packages with Preferences.toml files (added when building sysimages) in a file and run over that list for the cleaning ?

Do you suggest not to remove `SysimagePreferences.toml` files here? These are small files. So, it makes some sense.
https://github.com/petvana/AutoSysimages.jl/blob/195ac009cb226b2ad513cccfd3a0797cc6989aa4/src/AutoSysimages.jl#L482

**Comment by curio-sitas** [comment]
> > Maybe we could keep trace of all packages with Preferences.toml files (added when building sysimages) in a file and run over that list for the cleaning ?
> 
> Do you suggest not to remove `SysimagePreferences.toml` files here? These are small files. So, it makes some sense.
> 
> https://github.com/petvana/AutoSysimages.jl/blob/195ac009cb226b2ad513cccfd3a0797cc6989aa4/src/AutoSysimages.jl#L482

I did not see that projects path was kept, my bad. Nevertheless adding a parameter to clean only a specific package can be interesting.

File: src/AutoSysimages.jl (hunk 2 above)
Comment (petvana) [reply src/AutoSysimages.jl:464]: Fixed by e0521507826e4ba60b12ec566746aa6999749237, thanks.

**Comment by petvana** [comment]
I've decided that `SysimagePreferences.toml` will be kept because they are tiny and can be versioned for some projects. Although this PR is not perfect, I'll merge that, and we can polish some details in the future.

**Pull request merged**